.github/workflows/ucai-integration-tests.yml

```
@@ -36,3 +36,27 @@ jobs:
       - name: Run tests
         run: |
           pytest integrations/langchain/tests
+
+    anthropic_test:
+        runs-on: ubuntu-latest
+        strategy:
+          matrix:
+            python-version: ['3.9', '3.10']
+        timeout-minutes: 20
+        defaults:
+            run:
+                working-directory: unitycatalog-ai
+        steps:
+          - name: Checkout code
+            uses: actions/checkout@v4
+          - name: Set up Python
+            uses: actions/setup-python@v5
+            with:
+              python-version: ${{ matrix.python-version }}
+          - name: Install dependencies
+            run: |
+              pip install .
+              pip install integrations/anthropic[dev]
+          - name: Run tests
+            run: |
+              pytest integrations/anthropic/tests
```

The format seems not correct
```suggestion
      runs-on: ubuntu-latest
      strategy:
        matrix:
          python-version: ['3.9', '3.10']
      timeout-minutes: 20
      defaults:
          run:
              working-directory: unitycatalog-ai
      steps:
        - name: Checkout code
           uses: actions/checkout@v4
        - name: Set up Python
           uses: actions/setup-python@v5
           with:
             python-version: ${{ matrix.python-version }}
        - name: Install dependencies
           run: |
             pip install .
             pip install integrations/anthropic[dev]
        - name: Run tests
           run: |
             pytest integrations/anthropic/tests
```

---

weird auto-tab applied in IDE?